<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialogCache</class>
 <widget class="QWidget" name="NewDialogCache">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="enabled">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>190</height>
    </size>
   </property>
   <property name="title">
    <string>Enable cache</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Number of sets:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="number_of_sets">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Block size:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="block_size">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Degree of associativity:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="degree_of_associativity">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Replacement policy:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="replacement_policy">
      <item>
       <property name="text">
        <string>Random</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Least Recently Used (LRU)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Least Frequently Used (LFU)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="write_policy">
      <property name="title">
       <string>Write policy</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="write_pol">
         <item>
          <property name="text">
           <string>Write Through</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Write Back</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="write_alloc">
         <property name="text">
          <string>Write Allocate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="access_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Access time (in cycles)</string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_read">
      <property name="text">
       <string>Read:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_write">
      <property name="text">
       <string>Write:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="access_write">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_burst">
      <property name="text">
       <string>Burst:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="access_burst">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="access_read">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
